File: C# + .NET Core + ASP.NET/Models/Aggregate.cs
using System;
using System.Text;

namespace apples.Models
{
    public class Aggregate
    {
        public Guid AggregateUuid { get; set; }
        private StringBuilder State { get; } = new StringBuilder();

        public Aggregate(Guid aggregateUuid)
        {
            AggregateUuid = aggregateUuid;
        }

        public void Apply(Event evt)
        {
            State.Append(evt.Data);
        }

        public string GetState()
        {
            return State.ToString();
        }
    }
}

File: JavaScript + Node.js + Express/src/aggregate.ts
import { Event } from "./entities/event";

class Aggregate {
  private aggregateUuid: string;
  private state: string = "";

  constructor(aggregateUuid: string) {
    this.aggregateUuid = aggregateUuid;
  }

  apply(event: Event): void {
    this.state += event.data;
  }

  getAggregateUuid(): string {
    return this.aggregateUuid;
  }

  getState(): string {
    return this.state;
  }
}

export { Aggregate };